src/com/jambit/onboarding2020/tbrpg/domain/Room/BossRoom.java

```diff
@@ -12,6 +13,13 @@
 
 public class BossRoom extends AbstractRoom {
 
+    Player player = Player.getPlayerInstance();
+
+    GameInput in = new GameInput(new InputStreamReader(System.in));
+    Enemy enemy = new Enemy(); // Name übergeben von Enemy
+    Random random = new Random();
+    Weapon equippedWeapon = player.getEquippedWeapon();
```

Comment:
Man muss hier nicht equippedWeapon übergebn, da du einfach in der Methode ds Weapon abfragen kannst